Add model event and region.

erasmomontes opened the pull request:
To solve issues #1  add an application named
'events' in which I add the models related to the events,
these are 'Event' and "Region".

Also add the models to the admin of the mentioned application.

Commit:
Add model event and region.

To solve part of issues # 1 add an application named
'events' in which I add the models related to the events,
these are 'Event' and "Region".

Also add the models to the admin of the mentioned application.

xpostudio4 commented on comunidaddo/events/models.py:
```diff
@@ -0,0 +1,79 @@
+
+class Event(models.Model):
+
+    # Fields
```
I don't think the note fields is neccesary. By default almost all the elements state that.

xpostudio4 commented on comunidaddo/events/models.py:
```diff
@@ -0,0 +1,79 @@
+        ordering = ('-created',)
+
+    def __unicode__(self):
+        return u'%s' % self.slug
```
It is important to be consistent on the string formatting that we are going to use. Please chose one.

erasmomontes replied:
Ok, what if NameEntity: Principal Field ?

xpostudio4 commented on comunidaddo/events/models.py:
```diff
@@ -0,0 +1,79 @@
+    slug = extension_fields.AutoSlugField(populate_from='name', blank=True)
+    created = models.DateTimeField(auto_now_add=True, editable=False)
+    last_updated = models.DateTimeField(auto_now=True, editable=False)
+
```
The standard spacing for Django is two spaces between classes and one space between methods please adjust accordingly.

erasmomontes replied:
My bad, dont active the pylint in the edictor.

xpostudio4 commented on config/settings/base.py:
```diff
@@ -71,6 +71,7 @@
 LOCAL_APPS = [
     'comunidaddo.users.apps.UsersAppConfig',
+    'comunidaddo.events.apps.EventConfig',
     # Your stuff: custom apps go here
```
Please delete this comment on the line.

xpostudio4 commented on comunidaddo/events/admin.py:
```diff
@@ -0,0 +1,33 @@
+        'start_date', 'end_date', 'publish',
+        'publish_date', 'address', 'country', 'city']
+    readonly_fields = [
+        'slug', 'created', 'last_updated',
```
Why are all those fields read only?

erasmomontes replied:
Because they are fields generated by the system and I think it would not be good if they are changed by the admin because if that were the case the field loses credibility.

Example: if I create an event at a time X and then that time is modified, that data is false because in fact it was not created at that time, the same happens with the last_update field.

Commit:
Fixed code style and erase comment

With that changes fixed some errors of code style
and correct other point mentions in the review.
